.cloud-build/execute_changed_notebooks_helper.py

```diff
@@ -447,25 +447,5 @@ def process_and_execute_notebooks(
         # Raise error if any notebooks failed
         if not all([result.is_pass for result in results_sorted]):
             raise RuntimeError("Notebook failures detected. See logs for details")
-
```

Comment:
Removed this part for 2 main reason:
1. Adding another condition to the if statement at line 356 for python versions might make sense for now, but if we want to add more features in future we have to make that if statement conditions longer and longer. This way, every notebook is ran the same way no matter how many notebooks are found.
2. Better readability. First, when I read the code I was confused that we have 2 different process based on the fact that we have 1 notebook and more than 1. I think a separate process for want notebook makes changes more complicated.

Reply:
The separate process for a single notebook is needed as it's the only way to show results in the public logs (without having users go through a secondary download of executed notebooks).

Feel free to refactor but we need that option which as 99% of use cases.